Formula/p/porter.rb

```diff
@@ -0,0 +1,27 @@
+class Porter < Formula
+  desc "App artifacts, tools, configs, and logic packaged as distributable installer"
+  homepage "https://porter.sh"
+  url "https://github.com/SpiffyEight77/porter.git",
+    tag:      "v1.0.18",
+    revision: "bcc69cc41dce4d85611dca7306848d56b7b1a4b1"
+  license "Apache-2.0"
+  head "https://github.com/getporter/porter.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "mage" => :build
+
+  def install
+    system "mage", "-v", "BuildPorter"
+
+    bin.install "bin/porter"
+
+    generate_completions_from_executable(bin/"porter", "completion")
+
+    system "#{bin}/porter", "mixin", "install", "exec", "--version", "v#{version}"
```

Comment:
what does this do?

Reply:
> what does this do?

@chenrui333 

> We have a number of [mixins](https://porter.sh/mixins) to help you get started. Only the [exec mixin](https://porter.sh/mixins/exec/) is installed with Porter v1.0.0+, other mixins should be installed separately.

[Source Code](https://github.com/getporter/porter/blob/0427c4d73994f6d6d5af2b0b6b4b9ef850ddda19/scripts/install/install-mac.sh#L24)

This command is used to install the mixin feature for porter, because the official installation script will automatically install the [exec mixin](https://porter.sh/mixins/exec/).    🙇🏻‍♂️